README.md

```diff
@@ -527,6 +527,7 @@ You can see in which language an app is written. Currently there are following l
 - [iTerm 2](https://github.com/gnachman/iTerm2) - Terminal emulator for macOS that does amazing things. ![objective_c_icon]
 - [Kitty](https://github.com/kovidgoyal/kitty) - Cross-platform, fast, feature full, GPU based terminal emulator. ![c_icon] ![python_icon]
 - [OpenTerminal](https://github.com/es-kumagai/OpenTerminal) - App for macOS that opens a new Finder window and changes the current directory to the folder launched by the app. ![swift_icon]
+- [wallpapper](https://github.com/mczachurski/wallpapper) - wallpapper is a console application for creating dynamic wallpapers for Mojave. ![SwiftIcon]
```

Comment:
Please change from ![SwiftIcon] to ![swift_icon]

Reply:
Done